application/src/main/java/org/thingsboard/server/controller/BaseController.java

```diff
@@ -208,6 +209,9 @@ public abstract class BaseController {
     @Autowired
     protected DeviceService deviceService;
 
+    @Autowired
```

Comment:
No need to add this to the base controller. Add only to the DeviceConnectivityController.